snap/snapcraft.yaml

```diff
@@ -45,7 +45,7 @@ slots:
 parts:
   zookeeper:
     plugin: nil
-    source: https://github.com/canonical/central-uploader/releases/download/zookeeper-3.6.4-ubuntu0/zookeeper-3.6.4-ubuntu0-20231017125530.tgz
+    source: https://github.com/deusebio/central-uploader/releases/download/zookeeper-${SNAPCRAFT_PROJECT_VERSION}-ubuntu0/zookeeper-${SNAPCRAFT_PROJECT_VERSION}-ubuntu0-20231115215514.tgz
```

Comment:
Shouldn't this be stay as `canonical` though?

Reply:
yes, it will, once I'll start merging and release the final artifacts I will update the references. 

To test that artifacts work well with the charm, I built artifact on a feature branch on LP. The builds are then released to a forked repo, such that I can pull it to create a new snap to be released on a tmp channel `3/edge/feature-branch`, to be then used in the charm

Reply:
Why do you need a fork for that?
Why not use the Canonical one, and still release the same snap channel?

Reply:
After building an artifact from a feature branch in LP, we still need to test integration with snaps and charms before merging to the release branch and release it. 

I would like to release to [central-uploader](https://github.com/canonical/central-uploader/releases) only stable artifacts, without cluttering with "test" builds. Beside there are some constraints in the central uploader that would prevent you from overriding a `X.Y.Z-ubuntuW` release. Hence the fork.